zerver/forms.py

```diff
@@ -208,6 +216,20 @@ def clean_email(self) -> str:
         if realm.is_zephyr_mirror_realm:
             email_is_not_mit_mailing_list(email)
 
+        if settings.BILLING_ENABLED:
+            try:
+                check_license_limit(realm)
+            except LicenseLimitError:
+                message = f"Signup of {email} failed because of license limit. Please increase the licenses."
+                send_message_to_signup_notification_stream(
+                    get_system_bot(settings.NOTIFICATION_BOT), realm, message
+                )
+                raise ValidationError(
+                    _(
+                        "The organization is using all it's licenses and can't accept new members. Please contact the organization admin."
```

Comment:
Also here, who sees this message and where?

Can we tell them more specifically how to contact an admin?

Reply:
> Also here, who sees this message and where?

This would be shown to the user who is trying to sign up for the organization.

![Screenshot from 2021-05-25 16-36-22](https://user-images.githubusercontent.com/7190633/119487814-75338280-bd77-11eb-9def-ea4ae49536ce.png)

We also need to show a similiar message if they click on a confirmation/invite link from email when the licenses are fully used. As pointed by @mateuszmandera above.

Reply:
> Can we tell them more specifically how to contact an admin?

The only think we know about the admins is their email/names, but that's not something that we can show here I think. Do you have any ideas?

Reply:
How about:

`New members cannot join this organization because all Zulip licenses are in use. Please contact the person who invited you and ask them to increase the number of licenses, then try again.`